website/docs/developer/research/research-s3-bucket-policies.md

```diff
@@ -0,0 +1,156 @@
+---
+id: research-s3-practices
+title: S3 Future Direction/Best Practices
+description: Research notes on S3 best practices and some directions to take development.
+---
+
+## Overview
+
```

Comment:
Lots of good information, we may want to add some general guidelines in addition like the following:
- Enabling configurations to block all public access: https://docs.aws.amazon.com/AmazonS3/latest/userguide/access-control-block-public-access.html
- Using encryption on the buckets (we could use the default provided by AWS for free or make configurable if end users wanted something more): https://docs.aws.amazon.com/AmazonS3/latest/userguide/default-bucket-encryption.html
- Possibly create access through a VPC endpoint with the Cumulus OU: https://aws.amazon.com/blogs/architecture/choosing-your-vpc-endpoint-strategy-for-amazon-s3/
- Enforce object ownership by the ORCA account (this may go more with the ACL stuff though): https://docs.amazonaws.cn/en_us/AmazonS3/latest/userguide/about-object-ownership.html and https://aws.amazon.com/premiumsupport/knowledge-center/copy-s3-objects-account/
- Enforce TLS when uploading/copying, etc. https://docs.aws.amazon.com/config/latest/developerguide/s3-bucket-ssl-requests-only.html

Reply:
I believe ownership is covered by the ACL changes.

Reply:
Added possible method of enforcing TLS. Untested.

Reply:
Created https://bugs.earthdata.nasa.gov/browse/ORCA-454 for VPC

Reply:
Added comment and card for encryption. Looks like cross-account requires a customer-managed token.